blazar/api/v1/networks/service.py

```diff
@@ -93,3 +93,14 @@ def get_allocations(self, network_id, query):
         :type query: dict
         """
         return self.manager_rpcapi.get_allocations(network_id, query)
+
+    @policy.authorize('networks', 'get')
```

Comment:
Should we have a separate policy for listing properties? Maybe something to discuss upstream.

Reply:
Added